evadb/executor/create_function_executor.py

```diff
@@ -22,7 +22,9 @@
 from pathlib import Path
 from typing import Dict, List
 
+import numpy as np
 import pandas as pd
+from sklearn.metrics import mean_squared_error
```

Comment:
This breaks the documentation build and other usecases, the sklearn is not a default dependency in EvaDB. Either moves the import inside the forecast and use try_to_import_sklearn or add sklearn to the default dependency.

Reply:
Thanks @xzdandy! `mean_squared_error` is essentially a single line of code in `numpy`. I'll change it.